bin/ajax

```diff
@@ -371,6 +396,38 @@ def clean_abandoned(delete_live=False):
         return clean_abandoned(delete_live=delete_live)
 
 
+def clean_database(delete_live=False):
+    """
+    Remove entries from the database if the data is not on this host anymore
+    """
+    # We need to grep all the rundocs at once as simply doing one at the time (find_one)
+    # might get stuck as the query may yield the same result the next time it is called.
+    if not delete_live:
+        rds = run_coll.find({'bootstrax.state': 'done',
+                             'bootstrax.time':
+                                 {"$lt": now(-ajax_thresholds['wait_after_processing'])},
+                             'data.host': hostname})

```

Comment:
Do we want to add `status: 'transferred'` here as well, just to be explicit, or do we think it isn't necessary?